integration/confidential/lib.sh

```diff
@@ -129,16 +129,26 @@ disable_full_debug() {
 # Parameters:
 #	$1 - (Optional) file path to where save the current containerd's config.toml
 #
+# Environment variables:
+#	TESTS_CONFIGURE_CC_CONTAINERD - do not re-configure containerd if set to
+#	'no'. Default to 'yes'.
+#
 configure_cc_containerd() {
 	local saved_containerd_conf_file="${1:-}"
 	local containerd_conf_file="/etc/containerd/config.toml"
 
+	[ -n "$saved_containerd_conf_file" ] && \
+		cp -f "$containerd_conf_file" "$saved_containerd_conf_file"
+
+	# The test caller might want to skip the re-configure. For example, when
+	# installed via operator it will assume containerd is in right state
+	# already.
+	[ "${TESTS_CONFIGURE_CC_CONTAINERD:-yes}" == "yes" ] || return 0
```

Comment:
Hmm.  I'm not 100% sure about returning here.

The operator, for sure, will **not** do:
* https://github.com/kata-containers/tests/pull/5086/commits/50958ce8e715036a12867a8032e1b6ca519216d6#diff-d9fa72994d19f3acd66a01c6ee36939cbe4e2751b2f0c402bb9e39491385b837R163-R168
* https://github.com/kata-containers/tests/pull/5086/commits/50958ce8e715036a12867a8032e1b6ca519216d6#diff-d9fa72994d19f3acd66a01c6ee36939cbe4e2751b2f0c402bb9e39491385b837R174

If those are needed for something that's TEE specific, we should keep doing these before returning.

Reply:
@fidencio thanks for the comment. I removed that commit from this PR so that I can have a look with more time.